frappe/integrations/offsite_backup_utils.py

```diff
@@ -67,7 +67,23 @@ def get_latest_backup_file(with_files=False):
 
 	return database, config
 
+def take_files_backup():
+	"""Only zips and places public and private files in backup folder"""
+	from frappe.utils.backups import BackupGenerator
+
+	odb = BackupGenerator(
+		frappe.conf.db_name,
+		frappe.conf.db_name,
+		frappe.conf.db_password,
+		db_host=frappe.db.host,
+		db_type=frappe.conf.db_type,
+		db_port=frappe.conf.db_port,
+	)
 
+	odb.todays_date = now_datetime().strftime('%Y%m%d_%H%M%S')
+	odb.set_backup_file_name()
+	odb.zip_files()
```

Comment:
`zip_files` is deprecated. Use `backup_files` instead. 

ref: https://github.com/frappe/frappe/commit/b5252d11b95c8808dd825168ddead90826517281